clients/githubrepo/client.go

```diff
@@ -27,19 +28,21 @@ import (
 	"github.com/ossf/scorecard/clients"
 )
 
-const repoFilename = "./githubrepo.tar.gz"
+const repoFilename = "%v.%v.githubrepo.tar.gz"
```

Comment:
We don't need this to be repo-specific anymore. So maybe just:

```
const repoFilename = "githubrepo*.tar.gz"
```

And `ioutil.TempFile` will replace the `*` with the random string.

Reply:
> We don't need this to be repo-specific anymore. So maybe just:
> 
> ```
> const repoFilename = "githubrepo*.tar.gz"
> ```

sure, I can do that. Not sure why it's necessary though.

> And `ioutil.TempFile` will replace the `*` with the random string.

I left `%v.%v` to help with debugging. it does not hurt to have it, I think. wdut?